src/styles/stars.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+    background-position: center;
+    background-image: url(/src/images/star.svg);
+  }
+
+  &--4 &__star:nth-child(-n + 4) {
+    background-image: url(/src/images/star-active.svg);
```

Comment:
The same issue with the path to the image in `background-image` property. Please, change it to a relative path.